src/features/client-dashboard/components/ClientMattersGrid.tsx

```
@@ -0,0 +1,90 @@
+              <SkeletonLoader variant="text" width="w-32" />
+              <SkeletonLoader variant="text" width="w-24" />
+              <SkeletonLoader variant="text" width="w-20" />
+            </div>
+          ))}
+        </div>
+      ) : error ? (
+        <div className="mt-6 rounded-xl border border-card-border bg-card px-3 py-2 text-sm text-input-text">
+          {error}
+        </div>
+      ) : matters.length === 0 ? (
+        <p className="mt-6 text-sm text-input-placeholder">No matters yet. When your firm opens a matter for you, it'll show up here.</p>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)